node/src/components/storage.rs

```diff
@@ -556,7 +556,25 @@ impl Storage {
         match incoming.message {
             NetRequest::Deploy(ref serialized_id) => {
                 let id = decode_item_id::<Deploy>(serialized_id)?;
-                let opt_item = self.get_deploy(id).map_err(FatalStorageError::from)?;
+                let mut opt_item = self.get_deploy(id).map_err(FatalStorageError::from)?;
+
+                match &mut opt_item {
+                    Some(deploy) => {
```

Comment:
Since the `None` branch is empty:

```suggestion
                if let Some(deploy) = &mut opt_item {
```